docker/bin/upgrade-centos.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Usage: `./upgrade-centos.sh <BEFORE> <AFTER>`
+#
+# Builds an upgrade process for the puppet-agent module and tags as
+# "pa-dev:centos".
+#
+# Parameters:
+# - BEFORE: The puppet-agent package version that is installed prior to upgrade.
+# - AFTER: The puppet-agent package version that should exist after upgrade.
+set -e
+
+cd "$(dirname "$0")/../.."
```

Comment:
I'm surprised parsing this line by the shell works as expected...

Reply:
Yes, it allows execution of the file from any directory.